docs/solidity-by-example.rst

```diff
@@ -645,4 +645,499 @@ Safe Remote Purchase
 Micropayment Channel
 ********************
 
-To be written.
+Creating and verifying signatures
+=================================
+
+Signatures are used to authorize transactions,
+and they're a general tool that's available to
+smart contracts. In this chapter we'll use it to
```

Comment:
`prove (...) that a certain account approved a certain message` - isn't that the most general form of the purpose of a signature (and still, the sentence starts with "in this chapter...")?